src/api.rs

```diff
@@ -25,7 +25,7 @@ use std::sync::Arc;
 use std::collections::BTreeSet;
 use std::path::PathBuf;
 
-const LOOKAHEAD_FRAMES: u64 = 10;
+const LOOKAHEAD_FRAMES: u64 = 7;
```

Comment:
Out of curiosity, what is the significance of changing this to 7?

Reply:
Letting me test on a sample of 20 frames w/out having to edit it :) (I set it back to 10 now)